documentation/design-docs/dotnet-monitor.yaml

```diff
@@ -0,0 +1,457 @@
+swagger: "2.0"
+info:
+  description: "dotnet-monitor diagnostics agent"
+  version: "1.0.0"
+  title: "dotnet-monitor"
+host: "localhost:9999"
```

Comment:
Our current bindings, for reference:
- http://localhost:{port}
- https://localhost:{port}
port defaults to 52323 and 52324, but the port can be changed

Reply:
Use to be 😄

Reply:
Huh?

Reply:
I made a change in the code to 1) remove https hosting since it breaks on aks without a default certificate 2) make the binding address parameterizable. It still defaults to http://localhost:52323